package.json

```diff
@@ -13,7 +13,8 @@
     "test": "jest",
     "test:watch": "npm test -- --watch",
     "build": "npm run build:js && npm run build:sass",
-    "build:fonts": "mkdir -p dist/fonts && cp -R `node_modules`/font-awesome/fonts/* `node_modules`/npm-font-open-sans/fonts/* dist/fonts/",
+    "build:prod": "browserify --extension=jsx src/js/App.jsx -t [ babelify ] -g [ envify --NODE_ENV production ] -g uglifyify | uglifyjs --compress --mangle > dist/cloud-explorer.js",
+    "build:fonts": "cross-env mkdir -p dist/fonts && (cp -R ../font-awesome/fonts/* ../npm-font-open-sans/fonts/* dist/fonts/) || (cp -R node_modules/font-awesome/fonts/* node_modules/npm-font-open-sans/fonts/* dist/fonts/)",
```

Comment:
Why would node modules always be in `..`? It can be in `./node_modules` right?

Reply:
Hence the `||`. It first tries `..` in the case CE is used as a module and if it fails, tries `node_modules` for a standalone version.

Reply:
Han... :man_facepalming: